### PR TITLE
remove a superfluous variable

### PR DESCRIPTION
**Pull request by zhjwpku**
Since LOG_MSG has a DISPLAY_TXT option, we should use it to
reduce introduction of new variable and remove duplicate
echo messages.

Signed-off-by: Junwang Zhao <zhjwpku@gmail.com>

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/bin/lib/gp_bash_functions.sh [hunk 1]
```diff
@@ -230,86 +230,78 @@ LOG_MSG () {
...
-		exit 1
+	fi
+	exit 1
 	LOG_MSG "[INFO]:-End Function $FUNCNAME"
 }
 
 ERROR_CHK () {
 	LOG_MSG "[INFO]:-Start Function $FUNCNAME"
 	if [ $# -ne 3 ];then
-		INITIAL_LEVEL=$DEBUG_LEVEL
```
Comment (zhjwpku) [inline gpMgmt/bin/lib/gp_bash_functions.sh:297]: here is the main change, all other diffs are just tab/whitespace changes

**Comment by higuoxing** [comment]
Code changes look good! But I would suggest formatting the codes in a follow-up PR.

**Comment by zhjwpku** [comment]
> Code changes look good! But I would suggest formatting the codes in a follow-up PR.

sure, formatting codes stripped to another PR, thanks

**Review by higuoxing — APPROVED** [review]
LG! Thank you very much!

**Pull request merged**